src/test/java/com/wikia/webdriver/testcases/mercurytests/HeaderTest.java

```diff
@@ -0,0 +1,123 @@
+    PageObjectLogging.log("hero image", MercuryMessages.INVISIBLE_MSG, true);
+  }
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithInfoboxAndWithHeroImage")
+  public void mercury_header_checkElementsVisibilityWithInfoboxAndWithHeroImage() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_1)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    PageObjectLogging.log("header", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    PageObjectLogging.log("page title", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isHeroImageVisible(), "hero image is visible");
+    PageObjectLogging.log("hero image", MercuryMessages.VISIBLE_MSG, true);
+  }
+
+  @Test(groups = "mercury_header_checkElementsVisibilityWithInfoboxAndWithoutHeroImage")
+  public void mercury_header_checkElementsVisibilityWithInfoboxAndWithoutHeroImage() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_2)
+            .getHeader();
+
+    Assertion.assertTrue(header.isHeaderVisible(), "header is visible");
+    PageObjectLogging.log("header", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertTrue(header.isPageTitleVisible(), "page title is visible");
+    PageObjectLogging.log("page title", MercuryMessages.VISIBLE_MSG, true);
+    Assertion.assertFalse(header.isHeroImageVisible(), "hero image is not visible");
+    PageObjectLogging.log("hero image", MercuryMessages.INVISIBLE_MSG, true);
+  }
+
+  @Test(groups = "mercury_header_heroImageIsProperlyStyled")
+  public void mercury_header_heroImageIsProperlyStyled() {
+    Header header =
+        new ArticlePage()
+            .open(MercurySubpages.INFOBOX_1)
+            .getHeader();
+
+    String property = "background-color";
+    String value = header.getHeroImageCssValue(property);
+    Assertion.assertEquals(
+        value,
+        "rgba(255, 255, 255, 1)",
+        "hero image has proper background-color attribute"
+    );
+    PageObjectLogging.log(String.format("header's %s", property), value, true);
+
+    property = "background-position";
+    value = header.getHeroImageCssValue(property);
+    Assertion.assertEquals(
+        value,
+        "50% 50%",
+        "hero image has proper background-position attribute"
+    );
+    PageObjectLogging.log(String.format("header's %s", property), value, true);
+
+    property = "background-repeat";
+    value = header.getHeroImageCssValue(property);
+    Assertion.assertEquals(
+        value,
+        "no-repeat",
+        "hero image has proper background-repeat attribute"
+    );
+    PageObjectLogging.log(String.format("header's %s", property), value, true);
+  }
+
+  @Test(groups = "mercury_header_heroImageIsSquare")
+  public void mercury_header_heroImageIsSquare() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)